pwa/app/lib/utils.ts

```diff
@@ -11,6 +11,28 @@ export function parseDateString(date: string) {
   return new Date(date);
 }
 
+export function getEventDateString(event: Event, month: 'long' | 'short') {
```

Comment:
Should this return just `${startDateString}` if the event is 1 day?

Reply:
It does that 
```
if (startDate == endDate) {
  return endDateString;
}
```

There reason we return the end date rather than the start date is because the end date  includes the year.

Reply:
![image](https://github.com/user-attachments/assets/8311620c-9b31-4f89-b3bb-e266a20a0fdb)

Not sure what's going on here

Reply:
Fixed the equality check